application/actions/requests_fixtures_test.go

```diff
@@ -44,14 +44,19 @@ func createFixturesForRequestQuery(as *ActionSuite) RequestQueryFixtures {
 	org := userFixtures.Organization
 	users := userFixtures.Users
 
-	requests := test.CreateRequestFixtures(as.DB, 3, true)
+	org2 := models.Organization{Name: "org2", AuthType: AuthTypeGoogle, AuthConfig: "{}"}
+	as.NoError(org2.Save(as.DB))
+
+	requests := test.CreateRequestFixtures(as.DB, 4, true)
```

Comment:
Do you want to give it a specific UserID to avoid weird db fetching behavior?

Reply:
Good idea